### PR TITLE
Add support of project base dir from environment

### PR DESCRIPTION
**Pull request by toindev**
This change allows to use the environment variable SONAR_PROJECT_BASE_DIR to set the sonar.projectBaseDir property.

This is useful for monorepos, where you may want to use different settings for different sub-projets.

The change does nothing if the variable is not set.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ sonar.projectKey=<replace with the key generated when setting up the project on
 sonar.sources=.
 ```
 
+You can change the base directory by using the optional environment variable `SONAR_PROJECT_BASE_DIR`
```
Comment (simonbrandhof) [inline README.md]: Missing final dot!

Comment (toindev) [reply README.md]: Two dots for price of one ;)

**Comment by simonbrandhof** [comment]
I approved a bit too quickly. Why not declaring the basedir as [an input](https://help.github.com/en/articles/metadata-syntax-for-github-actions#inputs)? It's different than SONAR_HOST_URL because the latter is not supposed to be set by users. WDYT?

**Comment by toindev** [comment]
You're perfectly right, it makes way more sense. I'm rather new to Actions, so thanks for the tip.

**Comment by toindev** [comment]
Hi. It has been a while since the review. Is there something required of me before merging this PR?

**Pull request merged**